feature-libs/product-configurator/common/assets/translations/en/configurator-common.ts

```diff
@@ -100,6 +100,12 @@ export const configurator = {
       noAttributeText: 'Remove filter(s) to see Overview content',
       itemPrice: 'Item Price',
     },
+    overviewMenu: {
+      title: 'Menu',
```

Comment:
Is a generic title "Menu" helpful for users? Wouldn't it be better to have something like "Groups" or "Attribute Groups"? Yes, it's in the wireframes but does it make sense?

Reply:
Hmhh agree. @JayEmKay-8 What do you think? Or maybe 'Group structure'?

Reply:
Anthony is currently suggesting "Sections", but he wants to do a short hallway test before giving a definitive answer. So bear with me.